ACPs/77-reinventing-subnets/README.md

```diff
@@ -32,48 +32,202 @@ With no Elastic Subnets live on Mainnet, it is clear that Permissionless Subnets
 
 ## Specification
 
-### Separate Subnet Validators from Primary Network Validators
+At a high-level, Subnets will now be able to manage their validator sets external to the P-Chain by setting the blockchain ID and address of their "validator manager". This validator manager is able to send Avalanche Warp Messages to the P-Chain in order to modify the validator set registered for the Subnet. In response, the P-Chain will send Avalanche Warp Message back to validator managers to notify them of validator set changes that have taken effect. Subnet Validators will no longer be required to validate the Primary Network, which removes the requirement to stake 2000 $AVAX to be a Subnet Validator. Further, Subnet Validators will only be required to sync the P-Chain (not X/C-Chain) in order to track validator set changes and support cross-Subnet communication. To maintain a Subnet Validator on the P-Chain, a continuous fee must be paid in $AVAX as specified later in this ACP.
 
-Subnet Validators will only be required to sync the P-chain (not X/C-Chain) to track any validator set changes in their Subnet and to support Cross-Subnet communication via Avalanche Warp Messaging (see "Primary Network Partial Sync" mode introduced in [Cortina 8](https://github.com/ava-labs/avalanchego/releases/tag/v1.10.8)). The lower resource requirement in this "minimal mode" will provide Subnets with greater flexibility of validation hardware requirements as operators are not required to reserve any resources for C-Chain/X-Chain operation. Since Subnet Validators are no longer required to validate or sync the Primary Network, the 2000 $AVAX requirement can be removed. To meter the number of Subnet Validators on the network, a significantly lower, continuous fee is introduced later in this ACP.
+### P-Chain Warp Message Payloads
 
-#### Bootstrapping Subnet Nodes
+To support modifying a Subnet's validator set on the P-Chain via Warp messages, Warp message verification support will be added to the [`PlatformVM`](https://github.com/ava-labs/avalanchego/tree/master/vms/platformvm). Similar to Warp messages received by the C-Chain, in order for a Warp message to be considered valid by the P-Chain, at least 67% of the `sourceChainID`'s validator set must have partcipated in the aggregate BLS signature. 
```

Comment:
I believe it's 67% of the _weight_ rather than the number of validators. I re-worded it a bit to clarify that point and include that this threshold could be modified on a per-Subnet basis in the future.

```suggestion
To enable management of a Subnet's validator set externally to the P-Chain, Warp message verification will be added to the [`PlatformVM`](https://github.com/ava-labs/avalanchego/tree/master/vms/platformvm). For a Warp message to be considered valid by the P-Chain, at least 67% of the `sourceChainID`'s weight must have participated in the aggregate BLS signature. This is equivalent to the threshold set for the C-Chain. A future ACP may be proposed to support modification of this threshold on a per-Subnet basis.
```